pkg/model/osb/catalog.go

```diff
@@ -14,7 +14,14 @@
 
 package osb
 
+import _ "github.com/golang/glog" // import glog flags
```

Comment:
This comment isn't necessary

Reply:
I remember somewhere the guideline says there must be a comment for each unnamed imports. This is particular for importing glog flags so it will work with the package wide bazel test setup.

Reply:
Ah, you're right. I missed the underscore.